bin/tma_commander.py

```diff
@@ -290,6 +292,7 @@ async def command_loop(self):
                         await self.handle_command(cmd_name, args)
                 except Exception as e:
                     print(f"Command {cmd_name} failed: {e}")
+                    print(traceback.format_exc())
```

Comment:
I didn't know about the traceback module! In general I'd use logging.exception but I guess this works as well.

Reply:
CscCommander has no logger; it just prints directly to stdout. It would be smart to add one -- especially since I now time-tag all output from the CSC -- but that would have to be done in salobj. (I will also say that CscCommander is a quick and dirty "GUI" for CSCs. I don't want to spend a lot of time on it.)